go/vt/vttablet/tabletmanager/rpc_vreplication_test.go

```diff
@@ -60,6 +60,8 @@ const (
 	getAutoIncrementStep     = "select @@session.auto_increment_increment"
 	setSessionTZ             = "set @@session.time_zone = '+00:00'"
 	setNames                 = "set names 'binary'"
+	setNetReadTimeout        = "set @@session.net_read_timeout = 300"
```

Comment:
We were discussing this yesterday and the consensus was that we should not hardcode values since the actual value might vary based on the cluster environment. 

We can add flags for this as common vttablet flags similar to the query timeout PR. Your values seem fine as defaults to me.